app/views/posts/show.html.erb

```diff
@@ -19,5 +23,13 @@
   </li>
 <% end %>
   </ul>
+  <h3>Create a Comment</h3>
+<%= form_with(model: [@post, @post.comments.build], url: user_post_comments_path(@post.author, @post)) do |form| %>
+  <div class="field">
+    <%= form.label :text %>
+    <%= form.text_area :text %>
+  </div>
+  <%= form.submit "Create Comment" %>
+<% end %>
```

Comment:
- [ ] Please relocate this form for creating a comment to the `views/comments/new.html.erb` file.
- [ ] Also note that you would have to create a `new` method in your `CommentsController` which will serve up the view to create a new comment.
- [ ] Kindly note that you have to create a route in your routes file for the `new` action
```ruby
GET   /posts/:post_id/comments/new
```